open_learning_ai_tutor/intent_selector_test.py

```diff
@@ -0,0 +1,49 @@
+import pytest
+import json
+from open_learning_ai_tutor.constants import Intent
+from open_learning_ai_tutor.intent_selector import get_intent
+
+
+@pytest.mark.parametrize(
+    ("assesment_selections", "previous_intent", "intents"),
+    [
+        (["l"], [], [Intent.G_REFUSE]),
+        (["m"], [], [Intent.S_STATE, Intent.A_CURIOSITY]),
+        (["i"], [], [Intent.S_CALCULATION]),
```

Comment:
It might be worth moving these letter codes into constants since I can see this becoming a tripping point